src/hotspot/share/utilities/globalDefinitions.hpp

```diff
@@ -379,6 +379,9 @@ inline T byte_size_in_proper_unit(T s) {
 #define PROPERFMT             SIZE_FORMAT "%s"
 #define PROPERFMTARGS(s)      byte_size_in_proper_unit(s), proper_unit_for_byte_size(s)
 
+#define RANGEFMT                      "[" PTR_FORMAT " - " PTR_FORMAT "], (" SIZE_FORMAT " bytes);"
+#define RANGEFMTARGS(p1, size)    p2i(p1), p2i(p1 + size -1), size
```

Comment:
Please change this to display the typical "[..)" range (e.g. "[0xC0000000 - 0xE000000)" ). Using the [) notation makes it very clear that the last address is excluded.